legacy-sdk/whirlpool/tests/integration/v1-ix/two_hop_swap.test.ts

```diff
@@ -1867,7 +1909,22 @@ describe("two-hop swap", () => {
       }),
     ).buildAndExecute();
 
-    await sleep(2000);
+    warpClock(2);
+    // Wait for our events to be observed (LiteSVM delivers logs async)
+    await pollForCondition(
+      async () => ({
+        detectedSignatureOne,
+        detectedSignatureTwo,
+        eventVerifiedOne,
+        eventVerifiedTwo,
+      }),
+      (r) =>
+        r.detectedSignatureOne === signature &&
+        r.detectedSignatureTwo === signature &&
```

Comment:
not a huge deal, but i noticed the poll conditions & assertions are kind of redundant. would it be more appropriate to check that the detected signatures are just non-null and let the assertions handle the value equality?